### DISCUSSION TITLE
Option to disable startup text in editor window

### DISCUSSION THREAD

**Discussion by romilly** [mu-editor/mu#1467] [Ideas & New Features]
Feature request:

It would be great to have some way of disabling the
`# Write your code here :-)` message on start-up

The message can often be helpful, but it is a distraction if one is introducing mu in a book or video.

**Comment by tjguk** [comment]
ISTR that the notion of mode templates was introduced for the web mode. My suggestion would be to allow this to be overridden -- in this case to an empty file, but in another case to some locally-relevant text.

**Comment by romilly** [comment]
That would be ideal.

On Wed, 11 Dec 2019 at 14:36, Tim Golden <notifications@github.com> wrote:

> ISTR that the notion of mode templates was introduced for the web mode. My
> suggestion would be to allow this to be overridden -- in this case to an
> empty file, but in another case to some locally-relevant text.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mu-editor/mu/issues/973?email_source=notifications&email_token=AABEOPQICVBTLCV6DDO56MDQYD3GPA5CNFSM4JZP5562YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEGTLCHI#issuecomment-564572445>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABEOPVXF5EIQUGOCK5QFDLQYD3GPANCNFSM4JZP556Q>
> .
>


-- 
personal:@romillyc work:@rareblog skype:romilly.cocking web:
http://blog.rareschool.com/
